LiveSplit.SuperMetroid.asl

```diff
@@ -21,10 +21,38 @@ startup
 	settings.SetToolTip("firstMissile", "Split on the first Missile pickup");
 	settings.Add("allMissiles", false, "All Missiles", "ammoPickups");
 	settings.SetToolTip("allMissiles", "Split on each Missile upgrade");
+	settings.Add("specificMissiles", false, "Specific Missile Packs", "ammoPickups");
+	settings.SetToolTip("ammoPickups", "Split on specific Missile Pack locations");
+	settings.Add("alphaMissiles", false, "Alpha Missile Pack", "specificMissiles");
+	settings.SetToolTip("alphaMissiles", "Split on picking up the Alpha Missile Pack");
+	settings.Add("chargeMissiles", false, "Charge Missile Pack", "specificMissiles");
+	settings.SetToolTip("chargeMissiles", "Split on picking up the Charge Missile Pack");
+	settings.Add("hjMissiles", false, "High-Jump Missile Pack", "specificMissiles");
+	settings.SetToolTip("hjMissiles", "Split on picking up the High-Jump Missile Pack");
+	settings.Add("waveMissiles", false, "Wave Missile Pack", "specificMissiles");
+	settings.SetToolTip("waveMissiles", "Split on picking up the Wave Missile Pack");
+	settings.Add("whseMissiles", false, "Warehouse Missile Pack", "specificMissiles");
+	settings.SetToolTip("whseMissiles", "Split on picking up the Warehouse Missile Pack");
+	settings.Add("mainStreetMissiles", false, "Main Street Missile Pack", "specificMissiles");
+	settings.SetToolTip("mainStreetMissiles", "Split on picking up the Main Street Missile Pack");
+	settings.Add("wsMainShaftMissiles", false, "Wrecked Ship Main Shaft Missile Pack", "specificMissiles");
+	settings.SetToolTip("wsMainShaftMissiles", "Split on picking up the Wrecked Ship Main Shaft Missile Pack");
 	settings.Add("firstSuper", false, "First Supers", "ammoPickups");
 	settings.SetToolTip("firstSuper", "Split on the first Super Missile pickup");
 	settings.Add("allSupers", false, "All Super Missiles", "ammoPickups");
 	settings.SetToolTip("allSupers", "Split on each Super Missile upgrade");
+	settings.Add("specificSupers", false, "Specific Super Missile Packs", "ammoPickups");
+	settings.SetToolTip("ammoPickups", "Split on specific Super Missile Pack locations");
+	settings.Add("earlySupers", false, "Early Supers Missile Pack", "specificSupers");
+	settings.SetToolTip("earlySupers", "Split on picking up the Early Supers Missile Pack");
+	settings.Add("phantoonSupers", false, "Phantoon Super Missile Pack", "specificSupers");
```

Comment:
Is it common to call them "Phantoon Supers"? I've usually heard Left Side Wrecked Ship.

Reply:
Could just be me, but I feel like it's fairly common in any%. Probably should specify for the sake of clarity across categories, though. I'll get all these fixed up and resubmit.